openml/flows/flow.py

```diff
@@ -132,6 +132,28 @@ def __init__(self, name, description, model, components, parameters,
 
         self.extension = get_extension_by_flow(self)
 
+    def __str__(self):
+        object_dict = self.__dict__
+        output_str = ''
+        id = '\n%-16s: %s\n' % ('Flow ID', object_dict['flow_id'])
+        version = '%-16s: %s\n' % ('Flow Version', object_dict['version'])
+        url = '%-16s: %s\n' % ('Flow URL',
+                               'https://www.openml.org/f/' + str(object_dict['flow_id']))
+        name = '%-16s: %s\n' % ('Flow Name', object_dict['name'])
+        description = '%-16s: %s\n\n' % ('Flow Description', object_dict['description'])
+
+        binary = ''
+        if object_dict['binary_url'] is not None:
+            binary = '%-16s: %s\n\n' % ('Binary URL', object_dict['binary_url'])
+
+        upload = '%-16s: %s\n' % ('Upload Date', object_dict['upload_date'].replace('T', ' '))
+        language = '%-16s: %s\n' % ('Language', object_dict['language'])
```

Comment:
Please remove the language, it is kind of unimportant (it is mostly English anyway).